conf/readme.md

```diff
@@ -0,0 +1,118 @@
+# Import Directories
+
+## What is the import directory for?
+
+The `import/` directory provides a way for you to change your config settings without the need to even touch the main `/conf/` and `/db/` files.
+
+By placing your custom entries into the `import/` directory within these two locations, your core files will not need to have any conflicts resolved when you update your server. You store your changes, and the rest is updated with rAthena.
+
+## How does this work?
+
+Think of "import" as in "override". Place only the settings you have changed in the import files, or settings you are "overriding".
+
+For example, when setting up a server there are always a few config settings that users would like to change in order for rAthena to suit their needs. The following example will show you how to use the `/conf/import/` directory correctly. (for `/db/import/` eamples, see [/db/readme.md](/db/readme.md))
+
+### Login Server
+---
+We want to use MD5 passwords and disable _m/f account creation methods.
```

Comment:
Maybe encapsulate the underscores to prevent italicization. IE:
... and disable `_m/_f` account creation methods.